README.md

```diff
@@ -5,68 +5,45 @@
+ [Durable Functions](https://docs.microsoft.com/en-us/azure/azure-functions/durable/durable-functions-overview) is a programming model for describing _stateful_ workflows in a serverless enviroment. Durable Functions empowers programmers to specify their workflows _as code_, meaning that developers can make good use of Python's idioms,
+ tools, libraries, and its overall programming ecosystem to implement complex systems with ease.
 
-* Define workflows in code. No JSON schemas or designers are needed.
-* Call other functions synchronously and asynchronously. Output from called functions can be saved to local variables.
-* Automatically checkpoint progress whenever the function schedules async work. Local state is never lost if the process recycles or the VM reboots.
+ 🐍  Find us on PyPi [here](https://pypi.org/project/azure-functions-durable/) 🐍   
+
+
+## More than just stateful Azure Functions
+Durable Functions is offered as an extension of [Azure Functions](https://docs.microsoft.com/en-us/azure/azure-functions/functions-overview). The extension manages state, checkpoints, and restarts for you. Python Durable Functions' advantages include:
+
+* Defining workflows as code. No JSON schemas or designers are needed.
+* Managing return values, exceptions, and control-flow using familiar Python constructs.
+* Scheduling serverless functions synchronously and asynchronously.
+* Automatically checkpointing progress whenever the function schedules async work. Local state is never lost if the process recycles or the VM reboots.
 
 You can find more information at the following links:
 
 * [Azure Functions overview](https://docs.microsoft.com/en-us/azure/azure-functions/functions-overview)
 * [Azure Functions Python developers guide](https://docs.microsoft.com/en-us/azure/azure-functions/functions-reference-python)
 * [Durable Functions overview](https://docs.microsoft.com/en-us/azure/azure-functions/durable/durable-functions-overview?tabs=python)
 
-A durable function, or _orchestration_, is a solution made up of different types of Azure Functions:
-
-* **Activity:** the functions and tasks being orchestrated by your workflow.
-* **Orchestrator:** a function that describes the way and order actions are executed in code.
-* **Client:** the entry point for creating an instance of a durable orchestration.
+## Core Concepts
 
-Durable Functions' function types and features are documented in-depth [here.](https://docs.microsoft.com/en-us/azure/azure-functions/durable/durable-functions-types-features-overview)
+A Durable Functions application is a solution made up of Azure Functions serving one of four distinct roles. These roles are outline below: 
```

Comment:
```suggestion
Durable Functions introduces four new types of functions to help build stateful serverless applications. These new function types are:
```

I hesitate to say a "Durable Functions application" and then say its made up of only these four types of functions, as you could have a mix of Durable functions and non-durable functions in an application.

Reply:
Completely rewrote the README to just re-use the same wording as in here: https://docs.microsoft.com/en-us/azure/azure-functions/durable/durable-functions-overview?tabs=csharp

So I removed this section :)